insight-engine/latest/using/sql/syntax.md

```diff
@@ -119,13 +119,20 @@ If you don’t specify a field the search runs against name, description, title,
 
 > **Note:** If cross locale is not configured for the field then an exception occurs.
 
-The list of the default supported types as declared in the `<alfresco_home>/solr4/conf/shared.properties` file:
+The list of default supported Alfresco properties is declared in the `<search_services_home>/solrhome/conf/shared.properties` file:
```

Comment:
Do we use the same `<search_services_home>/solrhome/...` path example in the Insight Engine and Search Services docs ?

Reply:
it seems that way yes. I did check that, and it was already there. So I re used it.